cic/addon/pre-requisites/secure

```diff
@@ -26,6 +30,38 @@ if [[ $trusty_host == "" && $security == "true" ]]; then
     echo "security : $security"
 fi
 
+## flash tos and vbmeta to partitions
+if [[ $security == "true" ]]; then
+    if [[ -z $tos_dev || -z $vbmeta_dev ]]; then
+        echo "Partition /tos[10MB] and /vbmeta[1MB] shoud be present"
+        echo "Please create them"
+        exit -1
+    else
+        tos_img_size=`du -b $AIC_WORK_DIR/tos.img | awk '{print $1}'`
+        vbmeta_img_size=`du -b $AIC_WORK_DIR/vbmeta.img | awk '{print $1}'`
+
+        if [[ $tos_img_size -gt `expr $tos_dev_sec_num\*512` ]]; then
+            echo "error: tos image size is bigger than tos partition size"
+            exit -1
+        fi
+
+        if [[ $vbmeta_img_size -gt `expr $vbmeta_dev_sec_num\*512` ]]; then
+            echo "error: vbmeta image size is bigger than vbmeta partition size"
+            exit -1
+        fi
+
+        tos_blk_count=`expr ${tos_img_size} / 512`
```

Comment:
do we need to plus one to make sure count would never down round?

tos_blk_count=`expr ${tos_img_size} / 512 + 1`

I checked expr division in shell, expr 15 / 10 = 1, it makes down round.

Reply:
for vbmeta.img, it is fixed 4096 bytes. tos.img is padding 4096 bytes align. For partitions, always 1M align. Should not need to add round here.

Reply:
Done. :)